client/account-recovery/controller.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import LostPasswordPage from 'account-recovery/lost-password';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+export function lostPassword( context, next ) {
+	context.primary = <LostPasswordPage basePath={ context.path } className="account-recovery__container" />;
```

Comment:
It feels like `account-recovery__container` is unnecessary here and also confusing since we don't have an account recovery component.

@mtias which do we prefer more?
```
.lost-password.main {
    max-width: 480px;
    font-size: 13px;
}
```
or 
```
.is-section-account-recovery .main {
    max-width: 480px;
    font-size: 13px;
}
```